exec/cnex/nstring.c

```diff
@@ -157,6 +202,17 @@ TString *string_appendString(TString *s, TString *ns)
     return s;
 }
 
+TString *string_fillString(TString *s, char *c, size_t count)
```

Comment:
This should be just `char c`, there's no reason for it to be a pointer to a character.